### DISCUSSION TITLE
Special clicks and presses

### DISCUSSION THREAD

**Discussion by FredericoGauz** [juliencrn/usehooks-ts#418] [Ideas]
Hi,

first of all, thank you for the wonderful library! I feel that it is very well written and documented.

I was wondering if there would be interest to add hooks for things like long clicks, keyboard presses and things like this. I have written a few of that in the past and thought that it would be nice to contribute them to the project but I don't want to create a pull request that is not compatible with the project goals and will never be accepted.

Please let me know if these would be of interest.

**Comment by juliencrn** [comment]
Hi, yes it's more than welcome! Thanks
